sql/99-patches.sql

```diff
@@ -193,5 +193,50 @@ $$
 $P$
 );
 
+-------------------------------------------------------------------------------
+-- LOL 3.21: Add ver_datum_code column to bde.crs_work
+-------------------------------------------------------------------------------
+
+PERFORM _patches.apply_patch(
+    'LOL 3.21: Add ver_datum_code column to bde.crs_work',
+    $P$
+DO $$
+DECLARE
+  v_isversioned BOOL;
+BEGIN
+
+  v_isversioned := false;
+
+  IF EXISTS (SELECT p.oid
+             FROM pg_catalog.pg_proc p, pg_catalog.pg_namespace n
+             WHERE p.proname = 'ver_is_table_versioned'
+             AND n.oid = p.pronamespace
+             AND n.nspname = 'table_version')
+  THEN
+    IF table_version.ver_is_table_versioned('bde', 'crs_work')
+    THEN
+      v_isversioned := true;
+    END IF;
+  END IF;
+
+  -- If table is versioned, use table_version API to add columns
+  IF v_isversioned
+  THEN
+      PERFORM table_version.ver_versioned_table_add_column(
+          'bde',
+          'crs_work',
+          'ver_datum_code',
+          'CHAR(4)'
```

Comment:
VARCHAR(4)? That's how the Landonline database defines it

Reply:
I think I used another field as reference which was mapped to CHAR, but ok to change (and changed now) as PostgreSQL size is exactly the same for both types. Note in this schema we've a mix of VARCHAR(x) and CHAR(x) usages.